src/Common/src/Interop/Windows/mincore/Interop.Idna.cs

```diff
@@ -19,8 +19,7 @@ internal partial class mincore
                                         char* lpUnicodeCharStr,
                                         int cchUnicodeChar,
                                         [System.Runtime.InteropServices.OutAttribute()]
-
-                                        char* lpASCIICharStr,
+                                        IntPtr lpASCIICharStr,
```

Comment:
Yeah, it was an MCG problem.  @natemcmaster and I found it independently.  I think the right things are happening for the tooling fix, but changing this now unblocks people with the current builds.